package/boot/uboot-layerscape/files/fsl_ls1012a-rdb-uEnv.txt

```diff
@@ -2,6 +2,7 @@ fdtaddr=0x8f000000
 loadaddr=0x81000000
 fdt_high=0xffffffffffffffff
 initrd_high=0xffffffffffffffff
```

Comment:
fdt_high/initrd_high should be removed because of introducing bootm_size.

Reply:
Done. Thank You.